.github/workflows/testoperator_dispatch.yml

```
@@ -33,7 +33,7 @@ on:
         description: 'Validate results?'
         required: false
         type: boolean
-        default: false
+        default: true
```

```suggestion
        default: false
```

This is an escape hatch until validation is enabled for everything. Enabling this now will likely make a lot of tests start failing.

Validation is enabled on a spicepod-by-spicepod basis. E.g: https://github.com/spiceai/spiceai/blob/trunk/tools/testoperator/dispatch/tpch/sf1/federated/duckdb.yaml